src/lib/Preview.js

```diff
@@ -693,6 +730,12 @@ class Preview extends EventEmitter {
         // Setup loading UI and progress bar
         this.ui.showLoadingIndicator();
         this.ui.startProgressBar();
+
+        this.setupLogger({
+            consoleLevel: this.options.consoleLevel,
+            appHost: this.options.appHost,
```

Comment:
What if your appHost and the host where you want to save your logs are different? As mentioned above I think this is too sophisticated for a v1.